### ISSUE TITLE
Tarea 7

### ISSUE THREAD

**Issue by davidamaro** [cecepassat/Fisica_Computacional_Cordero_Silis#1]
@cecepassat No encontré tu tarea 7 y el archivo que incluía dicho número no tenía ningún ejercicio resuelto.

Calificación: 0.

Si necesitas ayuda con el curso puedes enviarme el correo para fijar una reunión.

**Comment by cecepassat** [comment]
Sí, me atrasé unos días y después el display de mi computadora murió por lo que tuve que mandarla a arreglar y me atrasé todavía más.

Voy a intentar hacerla de todos modos para tener todos los programas sólo por completes.

Gracias, seguramente la necesitaré.

Nos vemos mañana en clase.

Buen día.

--Cédric Alexander Cordero Silis

De: David Amaro Alcalá
Enviado: lunes, 11 de septiembre de 2017 14:31
Para: cecepassat/Fisica_Computacional_Cordero_Silis
CC: Cédric Alexander; Mention
Asunto: [cecepassat/Fisica_Computacional_Cordero_Silis] Tarea 7 (#1)

@cecepassat No encontré tu tarea 7 y el archivo que incluía dicho número no tenía ningún ejercicio resuelto.
Calificación: 0.
Si necesitas ayuda con el curso puedes enviarme el correo para fijar una reunión.
